src/cdfmean.f90

```diff
@@ -115,10 +124,15 @@ PROGRAM cdfmean
      PRINT *,'                   partial steps.'
      PRINT *,'       [ -var ]  : also compute the spatial variance of cdfvar '
      PRINT *,'       [ -zeromean ] : create a file with cdfvar having a zero spatial mean.'
+     PRINT *,'       [ -out_suff OUT-file-suffix ] : specify SUFFIX (+ directory) of the output files'
+     PRINT *,'       [ -basins n_basins basin_names.. ] : specify basins in ',TRIM(cn_fbasins),' to be used'
```

Comment:
It could be good to find a way to pickup all the basin in the file by default (ie without having to specify the number of variable and the variable name. Maybe something like a choice between your option [ -basins n_basins basin_names.. ] or [-basins_default] with no extra arguments means we pick the NEMO basin file with all the default variable names. It is a suggestion if you have time, not a requirement.